pycontw/pycontw-frontend
# feat-call-for-proposal-2023

## Types of changes


* **New feature**

## Description

1.  Apply two-col-wrapper
2. Update 2023 content to the page.

## Steps to Test This Pull Request


1. Go to speaking/cfp
2. Check the content in English and Chinese
3. Check the style and see if fit the design on Figma

## Expected behavior

See Figma: https://www.figma.com/file/W3hdepQ7v7ZGYRP6QctBXH/PyCon-Taiwan-%E5%AE%98%E7%B6%B2%E8%A6%96%E8%A6%BA-2022?node-id=39%3A1327&t=bgd3UTBIgUXcPXbT-0

## Related Issue

#336 
## Additional context
